include/lemlib/devices/encoder/encoder.hpp

```diff
@@ -18,15 +19,15 @@ class Encoder {
          *
          * @return float angle rotated by the encoder, in radians
          */
-        virtual float getAngle() = 0;
+        virtual Angle getAngle() = 0;
 
         /**
          * @brief Get the angle rotated by the encoder since the last time it was checked, in radians
          *
          * @param update whether to update the last angle measured by the encoder. True by default
          * @return float angle rotated by the encoder, in radians
```

Comment:
```suggestion
         * @return Angle angle rotated by the encoder
```
update doxygen comment